tests/e2e/magic_functions/functions/py_read.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+import mgp
+
+
+@mgp.function
+def return_function_argument(ctx: mgp.FuncCtx, argument: mgp.Any):
+    return argument
+
```

Comment:
For me this function does not work:
```python
from typing import List
@mgp.function
def reverse_list(ctx: mgp.FuncCtx, first: List):
    return reversed(first)
```
I get `ValueError: Unsupported PyObject conversion` on running:
`return py_read.reverse_list([1,2,3,4]);`
I've tried the same with procedures and it works there. Not sure what's going on :thinking:

Reply:
I've tried this code, and get the `Unsupported conversion` (I am pretty sure that generators are not supported as the resulting type):

```
@mgp.read_proc
def reverse_list(ctx: mgp.ProcCtx, listarg: mgp.List[mgp.Any]) -> mgp.Record(list=mgp.List[mgp.Any]):
    return mgp.Record(list=reversed(listarg))
```

However I would like to know the example that worked for you?

Reply:
I am not even able to import the query modules if I don't specify the arguments for List objects type

Reply:
aha sry you are right, I've forgotten reversed returns a generator, and in procedures, I used list instead of the generator (returning just the argument for testing purposes)...